graphx/src/main/scala/org/apache/spark/graphx/PartitionStrategy.scala

```diff
@@ -32,7 +32,7 @@ trait PartitionStrategy extends Serializable {
 object PartitionStrategy {
   /**
    * Assigns edges to partitions using a 2D partitioning of the sparse edge adjacency matrix,
-   * guaranteeing a `2 * sqrt(numParts) - 1` bound on vertex replication.
+   * guaranteeing a `2 * sqrt(numParts)` bound on vertex replication.
```

Comment:
Note I'm changing the bounds given in the doc since they were never correct for non perfect squares.